command_mode.c

```diff
@@ -2585,6 +2611,7 @@ struct command commands[] = {
 	{ "version",               cmd_version,          0, 0,  NULL,                 0, 0          },
 	{ "view",                  cmd_view,             1, 1,  NULL,                 0, 0          },
 	{ "vol",                   cmd_vol,              1, 2,  NULL,                 0, 0          },
+	{ "mute",                  cmd_mute,             0, 0,  NULL,                 0, 0          },
```

Comment:
Sorry, did not spot this before: the list of commands is sorted, please keep it that way.